content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,46 @@
+# DVC Experiments Overview
+
+DVC Experiments are captures automatically by DVC when you [run them]. Each
+experiment creates and tracks a project variation based on the changes in your
+<abbr>workspace</abbr>. Experiments preserve the latest commit in the current
+branch (Git `HEAD`) as their parent or _baseline_.
+
+<details>
+
+### ⚙️ How does DVC track experiments?
+
+Experiments are custom [Git references] (found in `.git/refs/exps`) with a
+single commit based on `HEAD` (not checked out by DVC). Note that these commits
+are not pushed to Git remotes by default (see `dvc exp push`).
+
+</details>
+
+Experiments will have an auto-generated ID like `exp-bfe64` by default. A custom
```

Comment:
`auto-generated ID` is actually `auto-generated name`. We discussed this in another PR, considering there are too many different things that can be called ID, I think it might be easier for the reader just to call these _name_.

Reply:
Agreed. Updated @iesahin